src/common/FileSystem.cpp

```diff
@@ -122,21 +131,13 @@ template<> struct SerializeTraits<FS::LoadedPakInfo> {
 
 namespace FS {
 
-#ifdef BUILD_ENGINE
```

Comment:
Bad merge?

Reply:
Probably, now fixed. Btw that looks overly complicated… :-/